modules/bcf/app/views/bcf/issues/configure_unknown_statuses.html.erb

```diff
@@ -3,23 +3,15 @@
 
 <%= form_tag({ action: :configure_import }, multipart: true, method: :post) do %>
   <%= render 'import_options_hidden_fields' %>
-  <h3>Invalid topic status names found</h3>
+  <h3><%= t('bcf.bcf_xml.import.invalid_statuses_found') %></h3>
   <ul>
     <% @importer.aggregations.unknown_statuses.each do |name| %>
-      <li> <%= name.present? ? name : '(No status provided)' %></li>
+      <li> <%= name.present? ? name : "(#{t('bcf.bcf_xml.import.no_status_provided')})" %></li>
     <% end %>
   </ul>
-  <div class="form--field -full-width">
-    <div class="form--field-container">
-      <label class="option-label">
-        <input type="radio" name="import_options[unknown_statuses_action]" value="cancel" checked>
-        Recommended: Correct the data first and try another import.
-      </label>
-    </div>
-  </div>
-  <label class="option-label">
-    <input type="radio" name="import_options[unknown_statuses_action]" value="chose">
-    Import them all setting status to
+  <p>
+    <input type="hidden" name="import_options[unknown_statuses_action]" value="chose">
+    <%= t('bcf.bcf_xml.import.import_statuses_as') %>
     <% options = Status.all.collect { |status| [status.name, status.id] } %>
     <%= select_tag 'import_options[unknown_statuses_chose_ids][]', options_for_select(options),
```

Comment:
Please do not use `p` tags for something that is no paragraph and also no unstyled select fields and labels. We have a ruby helper tag `styled_select_tag` which can be used for the correct classes. Otherwise for example mobile styles won't be applied and changes made to our fields won't be applied later.
So it should look somehow like this: 
```
<div class="form--field">
       <%= styled_label_tag 'import_options[unknown_types_chose_ids][],  t('bcf.bcf_xml.import.import_types_as') %> %>
        <%= styled_select_tag('import_options[unknown_types_chose_ids][],
                             options,
                              container_class: '-slim') %>
      </div>
```

The same applies for the other view files.

Reply:
Thank you. The `<P>` tags "smelled" like bad style, when I was writing them. I will also try to use the `styled_select_tag` helper.

Reply:
I just pushed some changes that use notifcation boxes and form fields instead of plain P tags. However, I could not really see real changes to the rendering of the select box. Should I see changes?

Apart from that I had to introduce some ugly NBSPs and BRs for padding/margin. I bet there is a nicer way. Inline styles? Are ugly, too.

Reply:
The select field itself looks pretty much the same. So there is no big difference to see. 

The `nbsp` and `br` however are not necessary. You missed some classes, that is why it did not add the margin by itself. In detail, the label should be really a `label`-tag with the according class `form--label` and a `for` attribute, for accessibility. Further the container `form--field-container` is missing. 
The resulting HTML should look like this:

```
<div class="form--field">
    <input type="hidden" name="import_options[unknown_types_action]" value="chose">
    <label class="form--label" for="import_options[unknown_types_chose_ids][]">Set all these types to</label>
    <div class="form--field-container">
      <span class=" form--select-container -slim">
        <select name="import_options[unknown_types_chose_ids][]" id="unknown_types_chose_ids" class="form--select form--inline-select">
           <option value="6">Bug</option>
           <option value="2">Task</option>
           <option value="1">none</option>
        </select>
     </span>
  </div>
</div>
```